modules/virt-what-you-can-do-with-virt.adoc

```diff
@@ -23,7 +23,9 @@ An enhanced web console provides a graphical portal to manage these virtualized
 
 [IMPORTANT]
 ====
-When you deploy {VirtProductName} with {rh-storage}, you must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
+If you deploy {VirtProductName} with {rh-storage-first}, you must create a dedicated storage class for virtual machine disks.
+
+If you run Windows VMs, you might need to perform link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] to improve their performance..
```

Comment:
The concern I have about this wording is that we tell them to create a dedicated storage class for all VMs but then the guidance for how to create that storage class is targeted to Windows VMs.  There is no general guidance for all VMs.  I would suggest not making this change at all.  The main reason we recommend using a dedicated SC for all VMs is because we want to simplify the operation of the cluster.  In newer versions of OpenShift, ODF creates a special SC automatically and OCP-Virt will use it automatically be default.  In older cluster versions, it really only makes sense to create this sc manually to mitigate the rxbounce issue in Windows VMs (as documented in the linked KBase article).